### PR TITLE
Upgrade grpc and grpc-utils version

### PR DESCRIPTION
**Pull request by saxenakshitiz**

### CODE REVIEW

File: hypertrace-core-graphql-platform/build.gradle.kts [hunk 1]
```diff
@@ -23,18 +23,21 @@ dependencies {
...
+    api("io.grpc:grpc-api:1.42.0")
+    api("io.grpc:grpc-core:1.42.0")
+    api("io.grpc:grpc-stub:1.42.0")
+    api("io.grpc:grpc-context:1.42.0")
     api("com.fasterxml.jackson.core:jackson-databind:2.12.3")
     api("com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.12.3")
     api("com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.12.3")
     api("org.apache.commons:commons-text:1.9")
     api("io.opentelemetry:opentelemetry-proto:1.1.0-alpha")
+    api("com.google.code.gson:gson:2.8.9") {
```
Comment (aaron-steinfeld) [inline hypertrace-core-graphql-platform/build.gradle.kts:35]: This shouldn't be needed - are we still seeing issues?

Comment (saxenakshitiz) [reply hypertrace-core-graphql-platform/build.gradle.kts:35]: Let me remove and check

Comment (saxenakshitiz) [reply hypertrace-core-graphql-platform/build.gradle.kts]: Removed

Comment (saxenakshitiz) [reply hypertrace-core-graphql-platform/build.gradle.kts:35]: This is needed else snyk build fails

Comment (aaron-steinfeld) [reply hypertrace-core-graphql-platform/build.gradle.kts:35]: yeah, just looked why - it's the intersection of a couple reasons:
- GQL runs snyk on all configs, not just runtime classpath. That's due to a bug in snyk where filtering classpath requires a match in every project. The runtime classpath correctly resolves without this.
- The compile classpath doesn't, because the grpc utils are encapsulated in graphql - so they don't show up on the compile classpath of the modues, by design. gson however does, because certain modules use protobuf-java-util directly for logging.

tl;dr - makes sense to leave it

**Review by aaron-steinfeld — APPROVED** [review]
